pkg/storaged/pages.jsx

```diff
@@ -528,62 +528,19 @@ export const PageTable = ({ emptyCaption, aria_label, pages, crossrefs, sorted,
                 size = <StorageSize size={size} />;
         }
 
-        function onClick(event) {
-            if (!event || event.button !== 0)
-                return;
-
-            if (page.location)
-                cockpit.location.go(page.location);
-        }
-
-        function is_clickable(element) {
-            return element.classList.contains("pf-m-clickable");
-        }
-
-        function next_clickable_sibling(element) {
-            do {
-                const next = element.nextElementSibling;
-                if (next && is_clickable(next))
-                    return next;
-                element = next;
-            } while (element);
-
-            return null;
-        }
-
-        function previous_clickable_sibling(element) {
-            do {
-                const prev = element.previousElementSibling;
-                if (prev && is_clickable(prev))
-                    return prev;
-                element = prev;
-            } while (element);
-
-            return null;
-        }
-
-        function onRowKeyDown(event) {
-            const { code, target } = event;
-
-            if (target.nodeName == "TR") {
-                if (code == "Space" || code == "Enter") {
-                    if (page.location)
-                        cockpit.location.go(page.location);
-                    event.preventDefault();
-                }
-                if (code == "ArrowDown") {
-                    const next = next_clickable_sibling(target);
-                    if (next)
-                        next.focus();
-                    event.preventDefault();
-                }
-                if (code == "ArrowUp") {
-                    const prev = previous_clickable_sibling(target);
-                    if (prev)
-                        prev.focus();
-                    event.preventDefault();
```

Comment:
We probably want to keep these just so we can move up and down with arrow keys. Sorry again for not spotting this first time around, think I was too focused on the new code 

I suppose from a11y perspective that

1. User can use arrow keys to get to the row they want
2. User can use tab from that point on to select the links
3. User can open said links normally as they are links (so no extra code for us for this step)

Reply:
> We probably want to keep these just so we can move up and down with arrow keys.

I thought that now that we have regular links, we can let the browser to its usual tab navigation, no?

Reply:
Yeah in a way, but with previous implementation it worked with browser navigation too. It cycled between the row and the ellipsis menu

And since it does change how the application works I would still keep it in for now, particularly due to the items I listed. Should be helpful for a11y in not having to tab through all the rows (which is even longer now with this PR) if you just want to get to a specific row

Reply:
Just keeping the old code didn't work since rows can't be focused anymore, I think. I made it so that the arrow keys move two dimensional between the buttons in the table. Please try it out!